packages/project-management-automation/lib/tasks/add-milestone/index.ts

```diff
@@ -1,36 +1,41 @@
+/**
+ * External dependencies
+ */
+import type { getOctokit } from '@actions/github';
+import type { PushEvent } from '@octokit/webhooks-types';
+
 /**
  * Internal dependencies
  */
-const debug = require( '../../debug' );
-const getAssociatedPullRequest = require( '../../get-associated-pull-request' );
+import debug from '../../debug';
+import getAssociatedPullRequest from '../../get-associated-pull-request';
 
-/** @typedef {import('@octokit/request-error').RequestError} RequestError */
-/** @typedef {ReturnType<import('@actions/github').getOctokit>} GitHub */
-/** @typedef {import('@octokit/webhooks-types').EventPayloadMap['push']} WebhookPayloadPush */
+/**
+ * Type definitions
+ */
+type GitHub = ReturnType< typeof getOctokit >;
 
 /**
  * Number of expected days elapsed between releases.
- *
- * @type {number}
  */
 const DAYS_PER_RELEASE = 14;
 
 /**
  * Returns true if the given error object represents a duplicate entry error, or
  * false otherwise.
  *
- * @param {unknown} requestError Error to test.
+ * @param requestError Error to test.
  *
- * @return {boolean} Whether error is a duplicate validation request error.
+ * @return Whether error is a duplicate validation request error.
  */
-const isDuplicateValidationError = ( requestError ) => {
+const isDuplicateValidationError = ( requestError: unknown ): boolean => {
 	// The included version of RequestError provides no way to access the
 	// full 'errors' array that the github REST API returns. Hopefully they
 	// resolve this soon!
 	const errorMessage =
 		requestError &&
 		typeof requestError === 'object' &&
-		/** @type {{message?: string}} */ ( requestError ).message;
+		( requestError as { message?: string } ).message;
```

Comment:
Let us ensure that `message` exists instead of asserting it.

```suggestion
		'message' in requestError &&
		requestError.message;
```